swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/Resource.kt

```diff
@@ -105,20 +105,60 @@ interface Timestamped {
   val createTs: Long
 }
 
+interface MarkedResourceInterface : Identifiable {
```

Comment:
So, I'd probably change this to `MarkedResource` and then the current data class of `MarkedResource` to say... `MarkedResourceImpl` or `DetailedMarkedResource` or something.

Reply:
(Discussed offline). I'm gonna leave as is because `MarkedResource` is used everywhere. If it becomes annoying later on I'll change it.